rolling/distribution.yaml

```diff
@@ -6037,6 +6037,16 @@ repositories:
       url: https://github.com/ros2/rmw_dds_common.git
       version: rolling
     status: maintained
+  rmw_desert:
+    doc:
+      type: git
+      url: https://github.com/signetlabdei/rmw_desert.wiki.git
```

Comment:
Maybe @cottsay can chime in here, I recall that the doc stanza is trying to run the doc job on the buildfarm.  It generally expects that you are pointing to the source code with a `dox` file in the root.  I'm not sure this will work.

Reply:
That's correct.  This stanza should point to the source repository.